sdk/monitor/monitor-opentelemetry/test/internal/unit/logs/logHandler.test.ts

```diff
@@ -32,12 +34,12 @@ describe("LogHandler", () => {
     metricHandler = new MetricHandler(_config);
     handler = new LogHandler(_config, metricHandler);
     exportStub = sinon.stub(handler["_azureExporter"], "export").callsFake(
-      (logs: any, resultCallback: any) =>
+      (lgs: any, resultCallback: any) =>
```

Comment:
Just a nit, but I don't see the need for this to get renamed. Was this done automatically by the lint fix?

Reply:
I could've used a better name (maybe `logsInner`) but eslint was complaining that `logs` is also defined in outter scope and this one is shadowing it)